xdrdemo/cxx/tiering_server.h

```diff
@@ -7,37 +7,39 @@
 #include <xdrpp/arpc.h>
 #include "kvprot.hh"
 
-typedef struct _Client_Storage {
-    TierServerID servertoconnectidentification;
-    const char *porttoconnect;
+struct Client_Storage {
+    TierServerID server_id;
     xdr::unique_sock fd; 
     xdr::pollset *ps;
     xdr::rpc_sock *rpcsoc;
     xdr::arpc_client_tier<KVPROT1> *client;
     std::thread *thrd;
-} Client_Storage;
+};
 
 class KVPROT1_master {
+private:
   std::map<Key, Value> vals_;
+  std::map<TierServerID, Client_Storage*> next_tier_connections_;
 public:
-  std::string node_identifier;
-  std::map<TierServerID, Client_Storage*> NextTierConnections;
-
+  std::string node_name_;
   using rpc_interface_type = KVPROT1;
 
+  KVPROT1_master() : node_name_(DEFAULT_SERVER) {}
```

Comment:
We should probably add some documentation for why there's 2 constructors at some point but no big.